examples/platforms/k32w/src/flash.c

```diff
@@ -182,13 +163,19 @@ uint32_t utilsFlashWrite(uint32_t aAddress, uint8_t *aData, uint32_t aSize)
     }
 
 exit:
-    return result;
+    /* workaround for flash write issue */
```

Comment:
Can we provide a bit more detail on what the flash write issue is?

Reply:
Solved.